code/game/objects/machinery/overwatch.dm

```diff
@@ -53,6 +56,24 @@ GLOBAL_LIST_EMPTY(active_laser_targets)
 /obj/machinery/computer/camera_advanced/overwatch/attackby(obj/item/I, mob/user, params)
 	return
 
+/obj/machinery/computer/camera_advanced/overwatch/give_actions(mob/living/user)
+	. = ..()
+	RegisterSignal(user, COMSIG_MOB_CLICKON, .proc/attempt_spotlight)
+
+/obj/machinery/computer/camera_advanced/overwatch/remove_eye_control(mob/living/user)
+	. = ..()
+	UnregisterSignal(user, COMSIG_MOB_CLICKON)
+
+/obj/machinery/computer/camera_advanced/overwatch/proc/attempt_spotlight(datum/source, atom/A, params)
+	SIGNAL_HANDLER
+	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_ORBITAL_SPOTLIGHT))
+		to_chat(source, "<span class='warning'>Orbital spotlight cooling, please wait.</span>")
+		return
```

Comment:
Please include a check for the current orbital level. I would suggest `CLOSE_ORBIT` and `SKIM_ATMOSPHERE` only. 

Inverse Square law! make it stronger when closer.